src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,11 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.